### PR TITLE
Dapp tutorial

### LINKED ISSUE #14: Add docs on how to write a dApp
* Point to [name-app code](https://gtithub.com/CosmWasm/name-app) and [deployment](https://cosmwasm.github.io/name-app) for first example (once updated to CosmWasm v0.10.0 and CosmJS 0.22).
* Explain a bit of the code (and clean it up?)
* Maybe point to [starport](https://github.com/tendermint/starport) if they add CosmWasm support.

### PR DESCRIPTION
**Pull request by abefernan**
Closes #14

### CODE REVIEW

**Review by orkunkl — APPROVED** [review]
LGTM

File: learn/frontend-dapp/bootstrap-dapp.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+# Bootstrap dApp
+
+There are two approaches to bootstrap a new dApp: as a lerna package in the monorepo or as a standalone app.
+
+## Monorepo template
+
+With this approach, we'll create another lerna package in the `packages/` directory that will use the local `logic` and `design` packages as dependencies.
+
+For that, you only need to copy the `_template` directory into `packages/` and rename it to `balance-checker`.
```
Comment (ethanfrey) [inline learn/frontend-dapp/bootstrap-dapp.md]: Can you add some links there, and code samples? They should be able to have a running dApp going by the end of the tutorial and have a clue how to hack it. Ideally, add a page on how to modify this dApp to do something.

First, something like the quick start guide:

```sh
git clone https://github.com/CosmWasm/dApps.git
cd dApps
cp -r _template packages/balance-checker
yarn && yarn build-deps
cd packages/balance-checker
yarn start
```

Once they see this, they can go onto another page that shows them how to make some simple changes to it and then view those. I don't know - Point to a new network? Query a given contract address? Something else? It doesn't need to look pretty, but should give them a rough start.

**Review by ethanfrey — APPROVED** [review]
This is a good overview, but not (yet) a tutorial. Please address the comment about quick start commands and merge it. Then start work on another PR with some code samples. Nothing fancy, you can use https://docs.cosmwasm.com/learn/hijack-escrow/hack-contract.html as an example (it adds a super minor command), but this can be a bit simpler, as you assume people already know JS and React, so you just need to lead them through making a query and making a tx. And then displaying very simple UI elements for them.

Making a new issue for this

File: learn/frontend-dapp/bootstrap-dapp.md (hunk 1 above)
Comment (abefernan) [reply learn/frontend-dapp/bootstrap-dapp.md]: I'll add those commands, but

> They should be able to have a running dApp going by the end of the tutorial and have a clue how to hack it

makes me think that maybe you didn't see `learn/frontend-dapp/dapp-development.md`? It was hidden by GitHub (the usual "Load large diff").

If you saw it and you'd like me to extend the example dApp with TXs I think that would be a great idea, though.

Comment (ethanfrey) [reply learn/frontend-dapp/bootstrap-dapp.md]: I did not see this on the diff. Sorry

Happy if you merge this, and I will review the actual docs site. Adding sending tx would be very good

**Pull request merged**